src/main/kotlin/com/bugsnag/android/gradle/internal/AbstractSoMappingTask.kt

```diff
@@ -61,11 +61,7 @@ abstract class AbstractSoMappingTask : DefaultTask() {
         outputZipFile(process.inputStream, dst)
 
         val exitCode = process.waitFor()
-        if (exitCode != 0) {
-            throw IllegalStateException(
-                "Failed to generate symbols for $dst, objdump exited with code $exitCode"
-            )
-        }
+        check(exitCode == 0) { "Failed to generate symbols for $dst, objdump exited with code $exitCode" }
```

Comment:
Shouldn't this be `!=`?

Reply:
`check` is `if(!condition)` so it is equivalent to the previous code ("make sure exitCode == 0")

Reply:
Sorry - yes 🤦